client/src/main/java/io/pravega/client/stream/StreamConfiguration.java

```diff
@@ -61,16 +61,20 @@ public class StreamConfiguration implements Serializable {
     private final RetentionPolicy retentionPolicy;
     
     /**
-     * The duration after the last call to {@link EventStreamWriter#noteTime(long)} which the
-     * timestamp should be considered valid before it is forgotten. Meaning that after this long of
-     * not calling {@link EventStreamWriter#noteTime(long)} the writer will be forgotten.
+     * The duration after the last call to {@link EventStreamWriter#noteTime(long)} until which the
+     * writer would be considered valid. Meaning that after this long of not calling
+     * {@link EventStreamWriter#noteTime(long)} a writer would be considered invalid
+     * and would not be waited upon before generating subsequent watermarks.
```

Comment:
Can we avoid the use of the words "valid" and "invalid" because they have not been defined and may be assumed to mean something else by a user.

Reply:
fixed